### DISCUSSION TITLE
Memory leak in Tag component

### DISCUSSION THREAD

**Discussion by rppig42** [NG-ZORRO/ng-zorro-antd#5628] [Q&A] [answered]
### Reproduction link
[https://angular-9ujwgs.stackblitz.io](https://angular-9ujwgs.stackblitz.io)

### Steps to reproduce
1. Open Chrome dev-tool and record Performance
2. Click regenerate button for multiple times
3. run garbage collection and stop recording
3. Observe performance graph

### What is expected?
Nodes will be cleaned

### What is actually happening?
Nodes will never be cleaned
![image](https://user-images.githubusercontent.com/48581046/88793733-e47c2c80-d1cf-11ea-8dc1-5b249296c4a5.png)

| Environment | Info |
|---|---|
| ng-zorro-antd | 9.3.0 |
| Browser | Chrome latest |

**Comment by hsuanxyz** [comment, marked as answer]
https://github.com/angular/angular/issues/30728

```ts
@Component({
  selector: "nz-demo-tag-basic",
  animations: [trigger('noop', [])],
  template: `<nz-tag *ngFor="let tag of tagsList;">...</nz-tag>`
})
```

![image](https://user-images.githubusercontent.com/22736418/89144787-47c7df00-d581-11ea-8e62-fa91a36b1d5f.png)
